source/synthDrivers/sapi5.py

```diff
@@ -69,8 +68,12 @@ def __init__(self, synthRef: weakref.ReferenceType):
 		self.synthRef = synthRef
 		self._writtenBytes = 0
 
-	def ISequentialStream_RemoteWrite(self, pv, cb):
-		# out: pcbWritten
+	def ISequentialStream_RemoteWrite(self, pv: LP_c_ubyte, cb: int) -> int:
+		"""This is called when SAPI wants to write (output) a wave data chunk.
+		@param pv: A pointer to the first wave data byte.
+		@param cb: The number of bytes to write.
+		@returns: The number of bytes written.
```

Comment:
Please see the [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md) for more information.

```suggestion
		:param pv: A pointer to the first wave data byte.
		:param cb: The number of bytes to write.
		:returns: The number of bytes written.
```